### PR TITLE
make boto model file optional

### PR DESCRIPTION
**Pull request by danabens**
make boto model file optional

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -313,14 +313,15 @@ def docker_image(boto_model_file):
         pass
 
     if boto_model_file is None:
-        raise ValueError('boto_model_file cannot be None')
+        print('boto_model_file is None, using default model.')
```
Comment (l2yao) [inline tests/conftest.py]: Why we no longer want to raise error here?

File: tests/integ-jobs/test_track_from_training_job.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def wait_for_job(job, sagemaker_client):
         while True:
             response = sagemaker_client.describe_training_job(TrainingJobName=job)
             status = response['TrainingJobStatus']
+            print('Training Job status is {}'.format(status))
```
Comment (l2yao) [inline tests/integ-jobs/test_track_from_training_job.py]: I dont know if we want to change print to loggers, but it should def be part of another task

Comment (danabens) [reply tests/integ-jobs/test_track_from_training_job.py]: its possible to emit output with pytest using:
```
tox -- --capture=no

File: tests/conftest.py (hunk 1 above)
Comment (danabens) [reply tests/conftest.py]: the purpose of the boto_model_file argument is to allow the docker image to use an unreleased version of the API, in most cases we just want to use the default. I added this raise during an earlier refactoring, initially the behavior assumed the argument was passed and failed if not given.

File: tests/integ-jobs/test_track_from_training_job.py (hunk 2 above)
Comment (danabens) [reply tests/integ-jobs/test_track_from_training_job.py]: created a task to track

**Pull request merged**